spec/graphql/mutations/subscriptions/create_spec.rb

```diff
@@ -43,12 +44,16 @@
         input: {
           customerId: customer.id,
           planId: plan.id,
-          name: 'invoice display name',
+          name: 'name',
           externalId: 'custom-external-id',
           billingTime: 'anniversary',
           endingAt: ending_at.iso8601,
           planOverrides: {
             amountCents: 100,
+            charges: [
+              id: charge.id,
+              invoiceDisplayName: 'invoice display name'
+            ]
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.